src/consumer/consumer.py

```diff
@@ -100,11 +100,39 @@ def heartbeat_callback(ch, method, properties, body):
             services_last_timestamp[json_data["service"]] = json_data["timestamp"]
         except Exception as e:
             print(f"\33[31mError indexing to Elasticsearch: {e}\33[0m")
+    
+ 
+    def send_error_email(service, timestamp, error, status):
+        email_content = f"""<heartbeat xmlns:r="http://www.w3.org/2001/XMLSchema">
+        <service>{service}</service>
+        <timestamp>{timestamp}</timestamp>
+        <status>{status}</status>
+        <error>{error}</error>"""
+
+        try:
+            publish_to_rabbitmq(email_content)
+            print("Error email content published to RabbitMQ successfully.")
+        except Exception as e:
+            print(f"Error publishing error email content to RabbitMQ: {e}")
+
+    def publish_to_rabbitmq(email_content):
+        connection = pika.BlockingConnection(pika.ConnectionParameters(host=host, virtual_host=virtual_host, credentials=credentials))
+        channel = connection.channel()
```

Comment:
Er is all een connectie met RabbitMQ (van de heartbeat), better deze dan hergebruiken.